### ISSUE TITLE
ReferenceError: Tinytest is not defined

### ISSUE THREAD

**Issue by erniesg** [RocketChat/Rocket.Chat#2917]
Your Rocket.Chat version: (make sure you are running the latest)
Latest because I cloned this project before running 1 hour ago

I cannot build the app. This is the output on my Terminal window -

> `Chens-MacBook-Air:Rocket.Chat erniesg$ meteor
[[[[[ ~/Rocket.Chat ]]]]]                     

> => Started proxy.                             
=> Meteor 1.3.1 is available. Update this project with 'meteor update'.
=> Started MongoDB.                           
   Linking                                   |
WARNING: The output directory is under your source tree.

> I20160417-21:44:21.878(8)? Updating process.env.MAIL_URL
I20160417-21:44:24.474(8)? ufs: store created at 
W20160417-21:44:25.231(8)? (STDERR) 
W20160417-21:44:25.232(8)? (STDERR) /Users/erniesg/.meteor/packages/meteor-tool/.1.1.10.hemqj2++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:245
W20160417-21:44:25.232(8)? (STDERR) 						throw(ex);
W20160417-21:44:25.232(8)? (STDERR) 						      ^
W20160417-21:44:25.249(8)? (STDERR) ReferenceError: Tinytest is not defined
W20160417-21:44:25.249(8)? (STDERR)     at Template.hello.greeting (packages/percolate_synced-cron/synced-cron-tests.js:15:1)
W20160417-21:44:25.249(8)? (STDERR)     at /Users/erniesg/Rocket.Chat/.meteor/local/build/programs/server/packages/percolate_synced-cron.js:606:4
W20160417-21:44:25.250(8)? (STDERR)     at /Users/erniesg/Rocket.Chat/.meteor/local/build/programs/server/packages/percolate_synced-cron.js:674:3
W20160417-21:44:25.250(8)? (STDERR)     at /Users/erniesg/Rocket.Chat/.meteor/local/build/programs/server/boot.js:242:10
W20160417-21:44:25.250(8)? (STDERR)     at Array.forEach (native)
W20160417-21:44:25.250(8)? (STDERR)     at Function._.each._.forEach (/Users/erniesg/.meteor/packages/meteor-tool/.1.1.10.hemqj2++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160417-21:44:25.250(8)? (STDERR)     at /Users/erniesg/Rocket.Chat/.meteor/local/build/programs/server/boot.js:137:5
=> Exited with code: 8

Could you please help me build the app? I want to make a few changes to the UI but I can't even get it to build :(

**Comment by marceloschmidt** [comment]
Hi, sorry for the delayed response.
Is this still happening to you?

**Comment by mrsimpson** [comment]
Hi, same issue here. Environment is windows, if that may "help"

**Comment by marceloschmidt** [comment]
Hi @mrsimpson what's your version?

**Comment by mrsimpson** [comment]
Answering your question probably comes with explanation and solution:
I had plainly installed meteor (which is in 1.3 something). After that, I ran `meteor update`. What I thought to do was updating the dependencies to the current version. What I didn't know (but what @sampaiodiego explained to me) is that meteor also updates the configuration of the app (RocketChat) to use the updated meteor core. 
Re-cloned the repo, didn't run the update and I'm happy to be using it on 1.2-mode now.

Cheers, 
Oliver

P.s.: I just had a dependency-issue with a package not being updateable to being 1.3.x compatible (here, it was `percolate:synced-cron`. After I removed the `Tinytest`-references from the percolate:synced-cron-files (simply by commenting the test-lines), I could start it without further issues - but of course that left me with some kind of undefined state which I reverted now.

**Comment by erniesg** [comment]
Worked for me after I uninstalled then reinstalled with dependencies a few
times… Running this on Mac if this helps anyone!


Sent with Mixmax





On Tue, May 10, 2016 3:24 AM, Oliver Jägle notifications@github.com wrote:
Answering your question probably comes with explanation and solution:
I had plainly installed meteor (which is in 1.3 something). After that, I ran meteor update . What I thought to do was updating the dependencies to the current version.
What I didn't know (but what @sampaiodiego explained to me) is that meteor also updates the configuration of the app
(RocketChat) to use the updated meteor core.
Re-cloned the repo, didn't run the update and I'm happy to be using it on
1.2-mode now.

Cheers,
Oliver

P.s.: I just had a dependency-issue with a package not being updateable to being
1.3.x compatible (here, it was percolate:synced-cron . After I removed the Tinytest -references from the percolate:synced-cron-files (simply by commenting the
test-lines), I could start it without further issues - but of course that left
me with some kind of undefined state which I reverted now.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub



陈恩娇 Chen Enjiao (Ernie)
Ernie 's Mobile: +6581833039 | Work: http://graymatterglobal.com | Website: http:// ernie.sg | Facebook: http://fb.me/ernie1688 | LinkedIn: http://sg.linkedin.com/in/erniesg | WeChat ID: erniesg
